checkstyle-suppressions.xml

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0"?>
 
 <!DOCTYPE suppressions PUBLIC
-     "-//Checkstyle//DTD SuppressionFilter Configuration 1.0//EN"
-     "https://checkstyle.org/dtds/suppressions_1_0.dtd">
+  "-//Checkstyle//DTD SuppressionFilter Configuration 1.2//EN"
+  "https://checkstyle.org/dtds/suppressions_1_2.dtd">
 
 <suppressions>
+  <suppress files="." checks="IndentationCheck"/>
```

Comment:
@rrchai This rule disables the indent warning you encountered in a Java POJO. The warning was conflicting with the auto-formatter (Prettier).